log-router/values.yaml

```diff
@@ -50,3 +50,5 @@ reloaderResources: {}
   # requests:
   #  cpu: 100m
   #  memory: 128Mi
+
+updateStrategy: OnDelete
```

Comment:
I think users will be surprised by this update strategy: https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/ - they need to manually delete old pods for their helm-upgrade to five them the expected results.

Since `OnDelete` seems to be kubernetes 1.5 default, let's stick to the current defaults. Then, you can customize set the updateStrategy to satisfy your use case - what do you think?

Reply:
>Then, you can customize set the updateStrategy to satisfy your use case

Yes, right. Sorry for not sharing my thought..

```Rolling Update``` option  is added in kubernetes 1.6.
In my use case, damonset pod is automatically update for ```helm upgrade```, so I added ```updateStrategy``` in templates. 

However, as default behavior, I think that it is better to match kubernetes default, and I set it as ```OnDelete``` in values.yaml.